pkg/accounting/accounting.go

```diff
@@ -299,6 +303,56 @@ func (a *Accounting) Debit(peer swarm.Address, price uint64) error {
 	accountingPeer.lock.Lock()
 	defer accountingPeer.lock.Unlock()
 
+	cost := price
+
+	// see if peer has surplus balance to deduct this transaction of
+	if accountingPeer.surplus {
```

Comment:
how does surplus become true after a restart?

Reply:
it does not, currently
however, we could initialize it in getAccountingPeer? either set it to true by default, or make a store get to see if it has > 0 surplus balance